source/fundamentals/time-series.txt

```diff
@@ -0,0 +1,55 @@
+===========
+Time Series
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecols
+
+
+Overview
+--------
+
+In this guide, you can learn about time series collections in the MongoDB
+Node.js driver. 
+
+Create a Time Series Collection
+-------------------------------
+
+You should create a time series collection using the MongoDB Shell ({+mongosh+}). For detailed
+instructions on creating a time series collection using the MongoDB Shell, see our
+:manual:`MongoDB Manual entry on time series collections </core/timeseries-collections/#create-a-time-series-collection>`.
+
+.. important:: Use the MongoDB Shell For Administrative Commands
+
+   You should not use the Node.js driver to perform administrative commands on
+   your MongoDB deployment. Use the MongoDB Shell to perform administrative
+   commands, such as:
```

Comment:
The driver can do these things but do we have a policy of recommending making infra scripts in mongosh instead of using your driver of choice?

If there is a link or explanation to why you "should not" can we put that here?

Reply:
We don't have anything written yet, afaik. That will happen as a part of this now!

Reply:
Hey Neal, we talked about this in DevEd-DBX standup and @terakilobyte suggested we talk to Bernie Hackett about making a document that formalizes this policy in our docs. As far as I am aware, the precise reason why we have this stance is not stated in our docs, and explaining it may be a large enough unit of work to constitute a separate ticket. 

Would you be comfortable if I leave a comment saying "TODO: link to the page explaining admin commands in drivers" in the page and then have as an acceptance criteria in the "Drivers And Admin Commands Page" linking this note to that page?

Reply:
I'm not sure we should publish something that advises against doing something that our driver supports. It feels like it will imply: "Are we intending to remove this functionality or is it buggy compared to mongosh?". 

So I think yes we can move this work along but would it be okay to just omit this advice?

Reply:
We can remove it for now. After some discussion via Slack, I will be authoring a separate page that we can link to in the future on why we _recommend_ people perform administrative operations via the shell.

Reply:
Hi Neal and Nathan, I commented out the admonition, made the language around recommending `mongosh` a little less strong, and made "linking pages that depend on the Admin Ops in Drivers page" as part of the [acceptance criteria for the Admin Ops page ticket](https://jira.mongodb.org/browse/DOCSP-18605).  Please let me know if this fails to address anyone's concerns.